### PR TITLE
N2-112 Reorder fields.

### PR DESCRIPTION
**Pull request by jonholdsworth**
Alphabetical sorting added to Advanced Search field for:
* Topic
* Usage
* Location
* Strong handshape
Also FieldChoice sorting added on 'english_name' attribute so that all default areas including Admin site are sorted as well.

image

### CODE REVIEW

**Review by Dimithri — APPROVED** [review]

File: signbank/dictionary/forms.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ class GlossSearchForm(forms.ModelForm):
     trans_lang = forms.ModelChoiceField(required=False, empty_label=_('Choose language'), queryset=Language.objects.all())
 
     # Adding topics
-    semantic_field = forms.ModelMultipleChoiceField(label=_('Topic'), queryset=FieldChoice.objects.filter(field='semantic_field'),
+    semantic_field = forms.ModelMultipleChoiceField(label=_('Topic'), queryset=FieldChoice.objects.filter(field='semantic_field').order_by('english_name'),
```
Comment (joshmcarthur) [inline signbank/dictionary/forms.py:98]: @jonholdsworth do you need this `order_by` as well as the `Meta` class definition?

File: signbank/dictionary/models.py [hunk 2]
```diff
@@ -302,7 +302,7 @@ def __str__(self):
         return self.english_name
 
     class Meta:
-        ordering = ['field', 'machine_value']
+        ordering = ['field', 'english_name', 'machine_value']
```
Comment (joshmcarthur) [inline signbank/dictionary/models.py]:
```suggestion
        ordering = ['field', 'english_name']
```

Given that our machine values are randomly generated, I think we should be clear here that there is no utility in using the machine value for ordering.

File: signbank/dictionary/forms.py (hunk 1 above)
Comment (jonholdsworth) [reply signbank/dictionary/forms.py:98]: Yes. The Meta ordering does not affect the Advanced Search form.

File: signbank/dictionary/models.py (hunk 2 above)
Comment (jonholdsworth) [reply signbank/dictionary/models.py]: Absolutely. Commit done.

**Review by joshmcarthur — APPROVED** [review]

File: signbank/dictionary/forms.py (hunk 1 above)
Comment (jonholdsworth) [reply signbank/dictionary/forms.py:98]: Much thank!

**Pull request merged**